pkg/emlib/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=emlib
+PKG_URL=https://github.com/basilfx/emlib
```

Comment:
just wondering: as we use your fork of the emlib, couldn't we simply apply the patches directly to your fork and skip them in RIOT?

Reply:
That's fine. I'll rename it, remove the patches and fix the URL.